app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -711,10 +711,12 @@ def build_toolbar_hide_button(id)
       return false if MIQ_AE_COPY_ACTIONS.include?(id) && User.current_tenant.any_editable_domains? && MiqAeDomain.any_unlocked?
       case id
       when "miq_ae_domain_lock"
-        return true unless editable_domain?(@record)
+        return true unless @record.editable_properties? && !@record.contents_locked?
```

Comment:
I can never understand unless with more than one condition, especially if one of those conditions is negated.  😕